packages/Webkul/API/Http/Controllers/Shop/ResourceController.php

```diff
@@ -86,9 +87,15 @@ public function index()
      */
     public function get($id)
     {
-        return new $this->_config['resource'](
-            $this->repository->findOrFail($id)
-        );
+        if (Auth::user($this->guard)->id === (int) $id) {
```

Comment:
This method ( get($id) ) is used to get many things using the id like orders, shipments etc.
According to your logic, it will be work only for one id and it is users id then how can we get other data.